hardware/Netatmo.cpp

```diff
@@ -2114,21 +2137,21 @@ bool CNetatmo::ParseDashboard(const Json::Value& root, const int DevIdx, const i
 		if (rain < m_OldRainCounter[ID])
 			m_RainOffset[ID] += m_OldRainCounter[ID];
 
-		m_OldRainCounter[ID] = rain;
+		m_OldRainCounter[ID] = rain_1;
 		std::stringstream v;
 		v << rain_1;
 		v << ";";
 		v << rain_24;
 		std::string sValue = v.str().c_str();
                 ///Debug(DEBUG_HARDWARE, "(%d) %s (%s) [%s] rain %s %s %d %d", Hardware_int, str_ID.c_str(), pchar_ID, name.c_str(), sValue.c_str(), m_Name.c_str(), rssiLevel, batValue);
 		//UpdateValueInt(0, str_ID.c_str(), 0, pTypeRAIN, sTypeRAINByRate, rssiLevel, batValue, '0', v.str().c_str(), Name, 0, m_Name);
-		SendRainSensor(ID, batValue, rain_24 + rain_1, name, rssiLevel);
-		//SendRainSensorWU(ID, batValue, rain_24, rain_1, name + "- WU", rssiLevel);
+		//SendRainSensor(ID, batValue, m_RainOffset[ID] + m_OldRainCounter[ID], name, rssiLevel);
+		SendRainSensorWU(ID, batValue, rain_24, rain_1, name, rssiLevel);
```

Comment:
@JurgenLB , why switch (back) to WU? What 'issue' does that solve?

Reply:
when I use "SendRainSensor(ID, batValue, rain_24 + rain_1, " the daily data seems correct but the historical data is not.
the original code was "SendRainSensor(ID, batValue, m_RainOffset[ID] + m_OldRainCounter[ID], " but this result in a very odd number for the current rain. 

![image](https://github.com/user-attachments/assets/2f352187-790e-45c8-a37b-286e4212bdad)

because the WU sensor uses also rain_24 and rain_1 as parameter I think this will work better.

Reply:
The past 3 days the Domoticz panel and history are corresponding to the Netatmo website and history.
With using the.  `SendRainSensorWU(ID, batValue, rain_24, rain_1, name, rssiLevel)`

![image](https://github.com/user-attachments/assets/4349ad58-7071-4f2f-85fa-5d88e70918a4)